website/docs/language/resources/syntax.mdx

```diff
@@ -53,29 +54,25 @@ attributes the resource supports.
 
 ### Providers
 
-Each resource type is implemented by a [provider](/terraform/language/providers/requirements),
-which is a plugin for Terraform that offers a collection of resource types. A
-provider usually provides resources to manage a single cloud or on-premises
-infrastructure platform. Providers are distributed separately from Terraform
-itself, but Terraform can automatically install most providers when initializing
+A [provider](/terraform/language/providers/requirements) is a plugin for Terraform 
+that offers a collection of resource types. Each resource type is implemented by a provider. A
+provider provides resources to manage a single cloud or on-premises
+infrastructure platform. Providers are distributed separately from Terraform, 
+but Terraform can automatically install most providers when initializing
 a working directory.
 
-In order to manage resources, a Terraform module must specify which providers it
-requires. Additionally, most providers need some configuration in order to
-access their remote APIs, and the root module must provide that configuration.
+To manage resources, a Terraform module must specify the required providers, see 
+[Provider Requirements](/terraform/language/providers/requirements).
```

Comment:
```suggestion
To manage resources, a Terraform module must specify the required providers. Refer to 
[Provider Requirements](/terraform/language/providers/requirements) for additional information.
```
Per our style guide, use "refer to" instead of "see".